backend/apps/mentorship/api/internal/mutations/module.py

```
@@ -399,3 +419,70 @@ def update_module(self, info: strawberry.Info, input_data: UpdateModuleInput) ->
+                user.username,
+                exc_info=True,
+            )
+            raise PermissionDenied(msg) from err
+
+        if not module.program.admins.filter(id=mentor.id).exists():
+            msg = "Only program admins can delete modules."
+            logger.warning(
+                "Mentor '%s' attempted to delete module '%s' without admin permissions.",
+                user.username,
+                module_key,
+                exc_info=True,
```

Ruff (0.14.10)

`exc_info=` outside exception handlers

Remove `exc_info=`

(LOG014)